.github/workflows/all_solutions.yml

```diff
@@ -364,17 +364,17 @@ jobs:
         if: ${{ failure() }}
         uses: actions/upload-artifact@c7d193f32edcb7bfad88892161225aeda64e9392 # v4.0.0
         with:
-          name: integration-test-artifacts
+          name: integration-test-working-directory-${{ matrix.namespace }}
           path: |
             C:\IntegrationTestWorkingDirectory\**\*.log
             C:\IntegrationTestWorkingDirectory\**\*.config
           if-no-files-found: error
 
       - name: Archive Test Artifacts
```

Comment:
How about we combine the artifact upload in the previous step with this one and just upload a single artifact per test matrix namespace?

Reply:
You're reading my mind, I had the same thought a few minutes ago while looking at something else.  I'll give it shot.